client/apps/edit/components/admin/components/verticals_tags.jsx

```diff
@@ -0,0 +1,112 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { Col, Row } from 'react-styled-flexboxgrid'
+import { data as sd } from 'sharify'
+import Verticals from '../../../../../collections/verticals.coffee'
+import { AutocompleteInlineList } from '/client/components/autocomplete2/inline_list'
```

Comment:
Just a style thing, but it would be nice to keep import paths consistent by always using absolute!

Reply:
The coffeescript absolute paths fail for me in tests, so had to use relative here.

Reply:
Ooh interesting. @damassi does that seem familiar to you? I feel like there should be a way we can get the coffeescript paths to be absolute imports too.

Reply:
Unfortunately we can't do absolute imports with CoffeeScript since we're piping everything through babel (which only transforms .js)

Reply:
See https://github.com/artsy/positron/blob/master/.babelrc#L19